semantic/src/definitions/modules/checkbox.js

```diff
@@ -0,0 +1,831 @@
+/*!
+ * # Semantic UI - Checkbox
+ * http://github.com/semantic-org/semantic-ui/
+ *
+ *
+ * Released under the MIT license
+ * http://opensource.org/licenses/MIT
+ *
+ */
+
+;(function ($, window, document, undefined) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Shadowing of global property 'undefined'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5182945)